### PR TITLE
Added kickbots command

### PR DESCRIPTION
**Pull request by minipif**
Added a command for removing all bots.

### CODE REVIEW

File: src/modules/CommandList.js [hunk 1]
```diff
@@ -186,6 +186,21 @@ Commands.list = {
             }
         }
     },
+    kickbots: function(gameServer) {
+        for (var i = 0; i < gameServer.clients.length;) {
+            if (typeof gameServer.clients[i].remoteAddress == 'undefined') {
+                var client = gameServer.clients[i].playerTracker;
+                var len = client.cells.length;
+                for (var j = 0; j < len; j++) {
```
Comment (Luka967) [inline src/modules/CommandList.js]: This `for` loop isn't required, `FakeSocket.js` has a function `close` which removes player cells.

**Comment by Luka967** [comment]
If changed well enough can kick a set amount of bots.

**Comment by aaxxo** [comment]
This is great!

**Comment by Luka967** [comment]
@vram4 Close? This was implemented by some other pull request.

**Pull request closed without merging**